server/routes/user.ts

```diff
@@ -184,19 +188,26 @@ router.post('/import-from-plex', async (req, res, next) => {
     for (const rawUser of plexUsersResponse.MediaContainer.User) {
       const account = rawUser.$;
       const user = await userRepository.findOne({
-        where: { plexId: account.id },
+        where: { email: account.email },
```

Comment:
We need to keep looking up by `plexId` first, and then `email` if that fails. If the user changes their plex email, it should update their email in Overseerr.